eras/babbage/test-suite/src/Test/Cardano/Ledger/Babbage/TxInfo.hs

```diff
@@ -271,8 +267,8 @@ txInfoTestsV1 ::
   , BabbageEraTxBody era
   , Value era ~ MaryValue (EraCrypto era)
   , EraPlutusTxInfo 'PlutusV1 era
-  , Inject (ContextError (AlonzoEra (EraCrypto era))) (ContextError era)
-  , Inject (ContextError (BabbageEra (EraCrypto era))) (ContextError era)
+  , Inject (AlonzoContextError era) (ContextError era)
+  , Inject (BabbageContextError era) (ContextError era)
```

Comment:
Instead of having two constraints, we could use the `AlonzoContextError` constructor below to get rid of the first constraint. This would make the types a bit nicer at the cost of making the implementation a tiny bit noisier, but I'm also fine with keeping the two constraints.

Reply:
I did exactly that in the Babbage era. I was too lazy and didn't want to do it here :grin: 

I'll change, since you brought it up.

Reply:
Done